physionet-django/events/templates/events/event_entries.html

```diff
@@ -17,11 +17,23 @@
             <td>{{ participant.user.email }}</td>
             <td>{{ participant.user.is_credentialed }}</td>
             <td>
-                {% if participant.is_cohost %}
-                    <input type="checkbox" name="toggle-cohost-status"  event-slug="{{ event.slug }}" value="{{ participant.id }}" disabled checked>
-                {% else %}
-                    <input type="checkbox" name="toggle-cohost-status"  event-slug="{{ event.slug }}" value="{{ participant.id }}" disabled>
-                {% endif %}
+            {% if not event.has_ended %}
+              {% if participant.is_cohost %}
+                <form class="manage-cohost" action="{% url 'manage_co_hosts' %}">
+                    {% csrf_token %}
+                    <input type="text" name="event" value="{{ event.slug }}" hidden>
+                    <input type="text" name="participant" value="{{ participant.id }}" hidden>
+                    <input type="submit" value="Remove cohost" class="btn btn-sm btn-danger">
+                </form>
+              {% else %}
+                  <form class="manage-cohost" action="{% url 'manage_co_hosts' %}">
+                    {% csrf_token %}
+                    <input type="text" name="event" value="{{ event.slug }}" hidden>
+                    <input type="text" name="participant" value="{{ participant.id }}" hidden>
+                    <input type="submit" value="Make cohost" class="btn btn-sm btn-success">
+                </form>
+              {% endif %}
```

Comment:
From what I see only the submit button is different here. Maybe the if-statement could be moved deeper to not duplicate the code.

Reply:
I've made the changes here, but the entire template will have to be changed, as a lot of things have been changed in the inactive_participants, and will need to be changed accordingly in this template.

Reply:
I've made the changes and moved the if statement further in.